.travis.yml

```diff
@@ -80,12 +76,17 @@ matrix:
           env: DOWNSTREAM=aws-encryption-sdk
         - python: 2.7
           env: DOWNSTREAM=dynamodb-encryption-sdk OPENSSL=1.1.0i
+          # dynamodb-encryption-sdk tests fail on xenial for whatever reason
+          dist: trust
```

Comment:
This...shouldn't work?

Reply:
I bet trusty is the default on parse failure